code/script.js

```diff
@@ -218,71 +221,103 @@ const generateBoard = () => {
           <button class="filled-button small" onclick="guess('${person.name}')">Guess</button>
         </div>
       </div>
-    `
-  })
-}
+    `;
+  });
```

Comment:
Ah now I see, I guess u have some fancy formatting extension? that changes '' to "", and puts a semicolon after each row?